scripts/analyze_ws_staleness.py

```
@@ -0,0 +1,127 @@
+    if not db_path.exists():
+        print(f"Database not found: {db_path}")
+        return
+
+    conn = sqlite3.connect(str(db_path))
+    
+    # Check NTP samples
+    ntp_cur = conn.cursor()
+    try:
+        ntp_cur.execute("SELECT ts, server, offset_ms, rtt_ms FROM ntp_samples ORDER BY ts ASC")
+        ntp_rows = ntp_cur.fetchall()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)